File: visitapp/app/src/main/java/com/vistapp/visitapp/listeners/UpdateFragmentListener.java
package com.vistapp.visitapp.listeners;

import com.vistapp.visitapp.model.DoctorModel;

/**
 * Created by Santiago Cirillo on 15/05/2017.
 */

public interface UpdateFragmentListener {
    void updateAdapter(DoctorModel doctorModel);

    void itemChecked(DoctorModel doctorModel, boolean isChecked);

    void navigateToMaps(DoctorModel item);
}

File: visitapp/app/src/main/java/com/vistapp/visitapp/listeners/UpdateFragmentListener.kt
package com.vistapp.visitapp.listeners

import com.vistapp.visitapp.model.DoctorModel

/**
 * Created by Santiago Cirillo on 15/05/2017.
 */

interface UpdateFragmentListener {
    fun updateAdapter(doctorModel: DoctorModel)

    fun itemChecked(doctorModel: DoctorModel, isChecked: Boolean)

    fun navigateToMaps(item: DoctorModel)
}
